web-marketplace/src/components/organisms/LoginFlow/LoginFlow.tsx

```diff
@@ -32,6 +32,7 @@ type Props = {
   qrCodeUri?: string;
   createProject?: boolean;
   isConnectingRef?: React.MutableRefObject<boolean>;
+  buyCreditsFrom?: string;
 };
 
 const LoginFlow = ({
```

Comment:
Note: in `APP-204` in file `pages/BuyCredits/BuyCredits.Form.tsx` we'll need to update `<LoginFlow>` to:

```
<LoginFlow
  isModalOpen={isModalOpen}
  onModalClose={onModalClose}
  wallets={walletsUiConfig}
  modalState={modalState}
  # New prop:
  buyCreditsFrom={projectHref}
/>
```